src/models/Application.ts

```diff
@@ -0,0 +1,68 @@
+import { Schema, model, models } from 'mongoose'
+import type { User } from 'next-auth'
+
+enum Decision {
+  'Accepted',
+  'Not Accepted',
```

Comment:
You might consider calling this 'rejected' or 'denied' or something. I think that one word keywords are easier to check and deal with